startup.sh

```diff
@@ -233,6 +270,14 @@ function startContainers {
         startUniconfig
       ;;
 
+      withunistore)
+        echo -e "${INFO} Deploying Uniflow, Uniconfig, Unistore and Monitoring services"
+        startMonitoring
+        startUniflow
+        startUniconfig
+        startUnistore
```

Comment:
Shouldn't `startUnistore` come before `startUniflow`?

Because `startUniflow` contains these lines:
```
  if [[ "$(docker service ls --format {{.Name}} | grep fm_unistore)" != "" ]]; then
    echo -e "${INFO} Update Frinx-Frontent: enable L3VPN automation"
    export GM_UI_ENABLED=true
  fi
  ```
  
  Will the `fm_unistore` be running if we first start Uniflow?

Reply:
Not needed, because we configure frontend settings directly in unistore part. That provides a correct configuration in sequential/particular setup, where for example just unistore is selected for a startup.